packages/create-emotion/types/index.d.ts

```diff
@@ -32,7 +32,7 @@ export interface ArrayClassNameArg extends Array<ClassNameArg> {}
 
 export type ClassNameArg =
   | undefined | null | boolean | string
-  | { [key: string]: boolean }
+  | { [key: string]: boolean | undefined }
```

Comment:
Would this need to support all of these types as values?

Reply:
We may need `any`, as described in #746.

Reply:
while `any` is technically correct I doubt we should actually recommend it, typings give us power to restrict usages to sane ones

Reply:
yes it's better with `undefined | null | boolean | string`